linux/files/dhclient.conf

```diff
@@ -0,0 +1,91 @@
+{# Macro definition #}
+{%- macro render_key(section, key) -%}
+{%- if section[key] is defined and section[key]|length > 0 %}
```

Comment:
don't know if `section[key] is defined` is working as you expect, I would instead use `section.get(key)` that defaults to None.